doc/cookbook/source/references.bib

```diff
@@ -167,6 +167,5 @@ @inproceedings{basak2008kernel
 	author={J. Basak},
 	booktitle={2008 19th International Conference on Pattern Recognition},
 	title={A least square kernel machine with box constraints},
-	year={2008},
-	pages={1-4},
+	year={2008}
```

Comment:
cool thx!

Did you render this locally (the html output) and check that it all works ?

Reply:
Yes it works